app/styles/layout/_footer.scss

```
@@ -0,0 +1,149 @@
+.footer {
+    background: $bg-black;
+    .footer-top {
+        border-bottom: 1px solid $border-black-fading;
+        @include responsive(tablet) {
+            padding: 30px 0;
+        }
+        .footer-top__title {
```

```suggestion
    background: $bg-black;
    
    .footer-top {
        border-bottom: 1px solid $border-black-fading;
        
        @include responsive(tablet) {
            padding: 30px 0;
        }
        
        .footer-top__title {
```
Mỗi block cách nhau 1 dòng cho dễ quan sát nhé

---

```suggestion
    background: $bg-black;
    .footer-top {
        border-bottom: 1px solid $border-black-fading;
        @include responsive(tablet) {
            padding: 30px 0;
        }
        &__title {
```